### PR TITLE
Linkify http://localhost:4200 in README.

### PR DESCRIPTION
**Pull request by adjohnson916**

### CODE REVIEW

**Comment by adjohnson916** [comment]
I've signed it!

**Comment by adjohnson916** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -115,4 +115,7 @@ ember try ember-data-canary
 ### Running the FireBlog demo app
 
 * `ember server`
-* Visit your app at http://localhost:4200.
+* Visit your app at [http://localhost:4200][].
```
Comment (tstirrat) [inline README.md]: Thanks for this. But we've been using the inline link syntax throughout this file:

e.g.

```
See the [Firebase documentation](https://firebase.com/docs/web/libraries/ember/guide.html#section-without-ember-cli) for instructions on getting started.
```

**Comment by adjohnson916** [comment]
@tstirrat I'd assumed it might come in handy for re-use elsewhere in the doc, but regardless, I've force pushed a re-commit with inline link.

**Pull request merged**